packages/frontend/src/components/UserSettings/SocialLoginsPanel/index.tsx

```diff
@@ -22,7 +22,7 @@ const isIssuerTypeAvailable = (
 ) => {
     switch (issuerType) {
         case OpenIdIdentityIssuerType.GOOGLE:
-            return !!health.auth.google.oauth2ClientId;
+            return health.auth.google.enabled;
```

Comment:
oauth2ClientId can be used for gdrive, so instead we disable goggle auth on new variable (and more consistent with the rest of SSO)